sympy/parsing/tests/test_latex.py

```diff
@@ -242,6 +242,7 @@ def test_import():
     (r"\int x \, dx", Integral(x, x)),
     (r"\log_2 x", _log(x, 2)),
     (r"\log_a x", _log(x, a)),
+    (r"5^0 - 4^0", _Add(_Pow(5, 0), _Mul(-1, _Pow(4, 0))))
```

Comment:
@asmeurer 's comment [from gitter](https://gitter.im/sympy/sympy?at=60a80aad863ba917ce49dc66)
```
parse_latex isn't designed to do evaluate=False in its present form. 
```

But isn't that how we test things for `parse_latex`? All of the functions with `_` are basically unevaluated form of those.

Reply:
This is just testing that the output of parsing is the same as the sympy expression -- and in this case what is equivalent to the unevaluated expression is obtained.